.travis.yml

```diff
@@ -0,0 +1,7 @@
+deploy:
+  provider: pypi
+  user: bluerobotics
+  password:
+    secure: LBxDL7RlsRC0BLc+2PX7mdbI/BdCD7T/M7IOdLKpsgQ2GUYn2ONRps61bKAiOrl2k68weajD1E6Gl/lEoXtIF3NPt3JxRHlIBrpr3j+EPoG6sqKb5NahAoYX3j4qsNF17JMjTD+oT4EAnikwrSidHHbIt4RAYhtrKw0Br2Ep5MvaaFKOizAPV4bgPA5cni0IJ7C4nV7piGheLxgV6jCMyR9uBThDhqIuoZo8lnOdpwbc3/JMVKijAaJ27h5qpHCV9+SsfG7CYIA/vpafxBljYzcpJt3nKniA5JdPS/N+VXE0hvd9ejaMQY/o4goc8yFVoZFNCoogM7cP9731dzRT3YCJ/bv8hijlQTFU0JXKeHJQMySfpJRiy73k5fyQz5rkFvCfXy2ta2ej+T5aqd6jvJ/9lsHUPPIffPoIFDkwoLwBvkOwZ4oCNTVMqF95ey47asSm7HGragtdrR6sFht4M9xevDn1tndW8jK4bw4PRO7cljFnQLnUVH8eGCWQr8tiRvg9gawCezgHDeRGPVHJtnsN6JJDWWgxrBs5RLNog02AeRF48esVMY01ed5ti0gUCJlYYTrsqo1Zb356Ez/BZTEygRZhkTZ22RkVklbkAimsAeEGE/DL+umYq2PLza4NVL8KUMFuBiprRofuSh62H2xNEZ3miLxeXNxQdnSIEQ0=
```

Comment:
just to make sure, did you create this secure key with travis secure variable tool ?

Reply:
yep